src/screens/ModelPage/ModelPage.js

```diff
@@ -92,206 +61,42 @@ function ModelPage() {
-              selected={mapOption === MAP_FILTERS.STATE}
-            >
-              {stateName}
-            </MapMenuItem>
-          )}
-        </MapMenuMobileWrapper>
-        <MapContentInner>
-          <MapWrapper visible={mapOption === MAP_FILTERS.NATIONAL}>
-            <Map
-              hideLegend={true}
-              setMapOption={setMapOption}
-              setMobileMenuOpen={setMobileMenuOpen}
-            />
-          </MapWrapper>
-
-          {stateId !== MAP_FILTERS.DC && (
-            <CountyMapAltWrapper visible={mapOption === MAP_FILTERS.STATE}>
-              <CountyMap
-                fill={
-                  projections.primary
-                    ? projections.getAlarmLevelColor()
-                    : '#e3e3e3'
-                }
-                stateSummary={stateSummary}
-                selectedCounty={selectedCounty}
-                setSelectedCounty={fullFips => {
-                  const county = _.find(
-                    US_STATE_DATASET.state_county_map_dataset[stateId]
-                      .county_dataset,
-                    ['full_fips_code', fullFips],
-                  );
-
-                  goTo(
-                    `/us/${stateId.toLowerCase()}/county/${
-                      county.county_url_name
-                    }`,
-                  );
-                  setMobileMenuOpen(false);
-                  setSelectedCounty(county);
-                }}
-              />
-            </CountyMapAltWrapper>
-          )}
-        </MapContentInner>
-      </MapContentWrapper>
-    );
-  };
 
   return (
     <Wrapper>
-      {metaTags}
+      <AppMetaTags
+        canonicalUrl={`/us/${stateId.toLowerCase()}`}
+        pageTitle={`${stateName} Forecast`}
+        pageDescription={actionTitle}
+        shareTitle={actionTitle}
+        shareDescription={actionDescription}
+      />
       <ContentWrapper>
-        {renderHeader()}
-        {projections && !projections.primary ? (
-          <div>
-            <StyledNoResultsWrapper>
-              <StyledNoResults>
-                <div>
-                  We either weren't able to get data for {selectedCounty.county}
-                  , {selectedCounty.state_code}, or no cases have yet been
-                  confirmed.
-                </div>
-                <div>
-                  We’re unable to produce our model for this county until we
-                  have data. Check back soon.
-                </div>
-                <div style={{ marginTop: '1rem' }}>
-                  View projections for{' '}
-                  <span onClick={() => goTo('/us/' + stateId.toLowerCase())}>
-                    {stateName}
-                  </span>
-                </div>
-              </StyledNoResults>
-            </StyledNoResultsWrapper>
-            {renderMapContent()}
-          </div>
-        ) : (
-          [renderMainContent(), renderMapContent()]
-        )}
+        <SearchHeaderWrapper>
```

Comment:
We do a lot of these wrappers that seem like they should just be implementation details of the underlying component..   Maybe I'm missing something, but this should just be <SearchHeader ../>, no?

Reply:
ya i mean ideally but i'd rather not mess too much with the css (this pr is already a lot and i'm just trying to move towards a lil more components?